generate_current_permissions.py

```diff
@@ -82,7 +93,7 @@ def generate_chain_files(chain):
 
 
 def main():
-    for chain in w3_by_chain:
+    for chain in AddrBook.chain_ids_by_name.keys():
```

Comment:
i know this is just a draft, but im not sure about this; addr book holds a lot of chains that we dont consider production (ie optimism or goerli)

Reply:
I think the idea is to provide an RPC if one exists.  Maybe we could test it and throw an exception here and now if it does not.